docs/sources/panels-visualizations/configure-data-links/index.md

```diff
@@ -26,7 +26,7 @@ weight: 80
 
 # Configure data links
 
-Data links allow you to provide more granular context to your links. You can create links that include the series name or even the value under the cursor. For example, if your visualization shows four servers, you can add a data link to one or two of them. You can also link panels using data links.
+Data links allow you to link to other panels, dashboards, and external resources while maintaining the context of the source panel. You can create links that include the series name or even the value under the cursor. For example, if your visualization shows four servers, you can add a data link to one or two of them. You can also link panels using data links.
```

Comment:
@nmarrs - What about this? I've used this again [here](https://github.com/grafana/grafana/pull/86893/files#diff-c9ac8cb4f306982a9a04badfad45725245a480cc08e8fd7a0ef8def27418c914R7).

Reply:
Looks good! :shipit: